README.md

```diff
@@ -3177,19 +3159,6 @@ resource cleanup when possible.
     name = 'Bozhidar'
     ```
 
-  * **(Option B)** Prefer double-quotes unless your string literal
-    contains `"` or escape characters you want to suppress.
-
-    ```Ruby
-    # bad
-    name = 'Bozhidar'
-
-    # good
-    name = "Bozhidar"
```

Comment:
just FYI: http://viget.com/extend/just-use-double-quoted-ruby-strings

It's been shown that double quotes don't actually decrease performance. I would argue that anytime spent thinking or commenting (in code reviews) over double vs single quotes would be a waste and maybe we should just use double quotes by default?